package.json

```diff
@@ -29,7 +29,6 @@
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
-    "@farmos.org/farmos-olgm": "^1.7.0",
```

Comment:
Might need to re-run `npm install`

Reply:
This was true and I force-pushed an updated version of this change. The only difference is that it now includes the corresponding change to the `package-lock.json` file.